docs/sources/flow/reference/config-blocks/import.file.md

```diff
@@ -15,26 +15,32 @@ title: import.file
 
 {{< docs/shared lookup="flow/stability/beta.md" source="agent" version="<AGENT_VERSION>" >}}
 
-The `import.file` block imports custom components from a file and exposes them to the importer.
+The `import.file` block imports custom components from a file or a folder and exposes them to the importer.
```

Comment:
Same question as above.. do we really mean `folder` here? or is this actually a `directory`?